apps/tester-app/package.json

```diff
@@ -16,16 +16,21 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.23.1",
-    "react": "18.2.0",
-    "react-native": "0.74.3",
-    "react-native-svg": "15.3.0"
+    "react": "18.3.1",
+    "react-native": "0.76.0-rc.1",
+    "react-native-svg": "15.8.0-rc.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.9",
+    "@babel/core": "^7.25.2",
+    "@babel/plugin-syntax-typescript": "^7.25.4",
```

Comment:
is this needed?

Reply:
Yes, apparently from now on, it is 
<img width="578" alt="image" src="https://github.com/user-attachments/assets/8a9f302e-f13c-45fd-b9b3-0941f5c59e99">

Reply:
is this comming from upgrade helper? or just because rnsvg doesnt work without it?